### DISCUSSION TITLE
Use "record" for CQRS Queries and Commands

### DISCUSSION THREAD

**Discussion by mus-h** [jasontaylordev/CleanArchitecture#400] [Ideas]
Use "record" for CQRS Queries and Commands, it will be simpler and cleaner and less prone to mistake.

**Additional context**
![Before and after](https://user-images.githubusercontent.com/30037608/115197941-349e7480-a0fa-11eb-8d01-3d5e0ce6f107.png)

**Comment by jasontaylordev** [comment]
I think this is a good idea, I'll look to include it in a future release.
